src/containers/Review/ReviewPageWrapper.tsx

```diff
@@ -17,6 +20,24 @@ const ReviewPageWrapper: React.FC = () => {
     serialNumber,
   })
 
+  const [updateReviewResponse] = useUpdateReviewResponseMutation({
+    onCompleted: ({ updateReviewResponse }) =>
+      console.log('Success to update reviewResponse: ', updateReviewResponse?.clientMutationId),
```

Comment:
Do we need this console.log? (And if not, the `onCompleted` has nothing to do. ;))

Reply:
That's more for development purpose. In case we need to make sure the responses are correctly send to the server. I think we can remove once this is all solid